### DISCUSSION TITLE
Request for feedback: an introduction to conducting interactive cognitive experiments online

### DISCUSSION THREAD

**Discussion by Max-Lovell** [jspsych/jsPsych#2029] [Show and tell]
Hi All,

I've been bugging some of you on here for help with jsPysch recently so many thanks for all your help (particularly Becky and Josh)!

I was completely new to coding starting this project. As I struggled at first to find a comprehensive guide on how to conduct my own online experiment in the way required for my study, or a place that collated all the existing guides - and as I learnt to code and some webdev I made changes to the existing tutorials - I've made a guide on how to do so online. This guide is found here: https://users.sussex.ac.uk/mel29/online_experiments.html.

It's fairly geared towards members of my university at times as I assume that'll be the bulk of my traffic, but it should be of use to others too. It doesn't look amazing at the moment, but I've run out time to sink into this project so the information and HTML of the page will be improved in time (it looks markedly better on desktop!)

Please let me know if any of you have any suggested changes to the approach!

Thanks,

Max

**Comment by becky-gilbert** [comment]
Awesome, thanks for sharing @Max-Lovell! You're right that it's hard to find a comprehensive guide to this stuff, and I think many people have to piece things together from lots of different sources. 

You could mention JATOS (https://www.jatos.org/) as another option for hosting experiments. It's a free and open source server-side tool for managing online experiments. It does have to be set up on your own server (usually with a reverse proxy, like Nginx or Apache), but then it eliminates the need for the PHP scripting, and it includes some useful features for creating experiments and managing data.
